hack/create_dependabot.py

```diff
@@ -0,0 +1,105 @@
+import yaml
+import collections
+from pathlib import Path
+
+repo_path = Path(__file__).parents[1]
+
+print(repo_path.anchor)
```

Comment:
Can we move this into main() func as well?  Is there a must reason we have to keep it outside?

Reply:
the print was for debugging so I will remove it, the repo_path I think can be added to main